Source/EventFlow/Extensions/EventFlowOptionsAggregatesExtensions.cs

```diff
@@ -62,7 +64,9 @@ public static class EventFlowOptionsAggregatesExtensions
             this EventFlowOptions eventFlowOptions,
             IEnumerable<Type> aggregateRootTypes)
         {
-            var invalidTypes = aggregateRootTypes
+            var aggregateRootTypeList = aggregateRootTypes.ToList();
```

Comment:
@JC008 The `ToList()` is to limit the number of times the enumeration is processed. The basic input reads all assembly types, inspects its properties and determines if its of a specific base type. If the `ToList()` isn't there, then all that will be done twice.